doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -174,10 +175,12 @@ Doc. ID: UG-SS
 - [12 Audit Log](#12-audit-log)
   - [12.1 Changing the Configuration of the Audit Log](#121-changing-the-configuration-of-the-audit-log)
   - [12.2 Archiving the Audit Log](#122-archiving-the-audit-log)
-- [13 Back up and Restore](#13-back-up-and-restore)
-  - [13.1 Back up and Restore in the User Interface](#131-back-up-and-restore-in-the-user-interface)
+- [13 Backing up and Restoring Configuration](#13-backing-up-and-restoring-configuration)
```

Comment:
This should be changed back to:

- [13 Back up and Restore](#13-back-up-and-restore)

Reply:
Currently the link in the ToC doesn't work, because the title is different in ToC and document body. To avoid breaking existing links, it's better to use the old title.